stage2/modules/eventloop-animation/README.md

```diff
@@ -1,63 +1,68 @@
 # [Stage#2.](../../) Event Loop & Animation
 
-## В данном модуле студентам необходимо изучить:
+## Module Overview 📚
 
-#### Event Loop
+This module introduces students to the Event Loop and Animation concepts in web development. It aims to provide a comprehensive understanding of how JavaScript handles asynchronous events and the principles behind creating animations using CSS3. The module covers topics ranging from understanding the Event Loop to mastering CSS3 animations.
 
-- Event Loop
-- Call stack
-- Web APIs
-- Callback queue
+## Learning Objectives 🎯
 
-#### Animations
+Students will:
 
-- CSS3 Transitions
-- CSS3 Transforms
-- requestAnimationFrame
-- Performance issues
+- Understand the Event Loop mechanism in JavaScript, including its components like the Call Stack, Web APIs, and Callback Queue.
+- Grasp the concept of single-threaded execution and how JavaScript handles concurrency through asynchronous operations.
+- Learn about CSS3 Transitions and Transforms for creating animations.
+- Explore techniques such as requestAnimationFrame and addressing performance issues in animations.
 
-## Терминология, которую вам необходимо знать
+## Approximate Module Completion Time ⏱️
 
-- Event Loop
-- asynchronous
-- single-thread
-- concurrency
-- Non-blocking
-- Call stack
-- Web APIs
-- Callback queue
+- **4 hours**
 
-## Приблизительное время прохождения модуля
+## Theory 📖
 
-2 часа
+Students are encouraged to study the following resources:
 
-## Теория
+1. **Event Loop:**
 
-## Практика
+   - [Event Loop: Microtasks and Macrotasks](https://javascript.info/event-loop)
+   - [The Event Loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Event_loop)
+   - [Introduction to Web APIs](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Introduction)
 
-Вам необходимо пройти тест "Event Loop & Animation" в RS APP > Auto Test
+2. **Animations:**
 
-### Video
+   - [Web Animation Performance Fundamentals](https://www.freecodecamp.org/news/web-animation-performance-fundamentals/)
+   - [CSS and JavaScript animation performance](https://developer.mozilla.org/en-US/docs/Web/Performance/CSS_JavaScript_animation_performance)
 
```

Comment:
```suggestion
   - [CSS and JavaScript animation performance](https://developer.mozilla.org/en-US/docs/Web/Performance/CSS_JavaScript_animation_performance)
   - [Request Animation Frame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)

```
Please, add a link to https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame